client/src/containers/AssetDisplay/view.jsx

```diff
@@ -2,14 +2,49 @@ import React from 'react';
 import ProgressBar from '@components/ProgressBar';
 import { LOCAL_CHECK, UNAVAILABLE, ERROR, AVAILABLE } from '../../constants/asset_display_states';
 
+class AvailableContent extends React.Component {
+  render () {
+    const {contentType, sourceUrl, name, thumbnail} = this.props;
+    switch (contentType) {
+      case 'image/jpeg':
+      case 'image/jpg':
+      case 'image/png':
+      case 'image/gif':
+        return (
+          <img
+            className='asset-image'
+            src={sourceUrl}
+            alt={name}
+          />
+        );
+      case 'video/mp4':
+        return (
+          <video
+            className='asset-video'
+            controls poster={thumbnail}
+          >
+            <source
+              src={sourceUrl}
+            />
+            <p>Your browser does not support the <code>video</code> element.</p>
+          </video>
+        );
+      default:
+        return (
+          <p>Unsupported content type</p>
+        );
+    }
+  }
+}
+
 class AssetDisplay extends React.Component {
   componentDidMount () {
     const { asset: { claimData: { name, claimId } } } = this.props;
     this.props.onFileRequest(name, claimId);
   }
   render () {
-    const { status, error, asset: { claimData: { name, claimId, contentType, fileExt, thumbnail } } } = this.props;
-    const sourceUrl = `/${claimId}/${name}.${fileExt}`;
+    const { status, error, asset: { name, claimData: { claimId, contentType, fileExt, thumbnail, outpoint } } } = this.props;
+    const sourceUrl = `/${claimId}/${name}.${fileExt}?${outpoint}`;
```

Comment:
if this isn't being fixed before shipping, it needs to be filed, and it's a good practice to link the issue as an inline comment